src/_webrsp.m

```diff
@@ -273,10 +273,13 @@ N ARGS S ARGS("*")=$E(HTTPREQ("path"),2,9999)
  E  D W("Content-Type: application/json; charset=utf-8"_$C(13,10))
  ;
  ; Add CORS Header
- I $G(HTTPREQ("method"))="OPTIONS" D W("Access-Control-Allow-Methods: OPTIONS, POST"_$C(13,10))
- I $G(HTTPREQ("method"))="OPTIONS" D W("Access-Control-Allow-Headers: Content-Type"_$C(13,10))
- I $G(HTTPREQ("method"))="OPTIONS" D W("Access-Control-Max-Age: 86400"_$C(13,10))
- D W("Access-Control-Allow-Origin: *"_$C(13,10))
+ I $G(CORS("enabled"))="Y" D
+ . I $G(HTTPREQ("method"))="OPTIONS" D
+ . . D W("Access-Control-Allow-Credentials: "_CORS("credentials")_$C(13,10))
+ . . D W("Access-Control-Allow-Methods: "_CORS("method")_$C(13,10))
+ . . D W("Access-Control-Allow-Headers: "_CORS("header")_$C(13,10))
+ . . D W("Access-Control-Max-Age: 86400"_$C(13,10))
```

Comment:
this should probably be an option as well.

Reply:
Done